main.html

```diff
@@ -40,35 +42,120 @@
       </div>
+
+ 
+
     </section>
 
     <section class="content p-3 p-lg-5 d-flex flex-column" id="about">
       <div>
         <h2 class="mb-5">About</h2>
-        Once upon a time, I woke up on time. im a loser
+        Hello there! I'm Kelly and I'm an engineering student at the University of Waterloo. My design experience ranges from PCB design to power distribution (grammar?). I'm interested in ???. Currently looking for job opportunities for fall 2019. pLS hire.
       </div>
     </section>
 
-
+<!---PROJECTS--->
     <section class="content p-3 p-lg-5 d-flex flex-column" id="projects">
       <div>
         <h2 class="mb-5">Projects</h2>
-        I have a dog.
+
+        <div class="container">
+	        <!--Projects row 1--->	
+	        <div class="row no-gutters">
+		        <!--Driver Display project-->
+		        <div class='my-project'>
+		        	<div class="col-md-6"></div>
+		        	<a href="s" data-toggle="modal" data-target="#DriverDisplayModal">
+		        	<img class="img-fluid" src="images/display4.png" alt="">	
+		        	</a>
+		        </div><!--END Driver display project-->
+
+		        <!--Driver Controls project-->
+		        <div class='my-project'>
+		        	<div class="col-md-6"></div>
+		        	<a href="s" data-toggle="modal" data-target="#DriverDisplayModal">
+		        	<img class="img-fluid" src="images/ms12.jpg" alt="">	
+		        	</a>
+		        </div><!--END Driver Controls project-->
+	    	</div><!---END projects row1--->
+
+	    	<!--Projects row 2--->
+	    	<div class="row no-gutters">
+		         <!--Onion Omega project-->
+		        <div class='my-project'>
+		        	<div class="col-md-6"></div>
+		        	<a href="s" data-toggle="modal" data-target="#DriverDisplayModal">
+		        	<img class="img-fluid" src="images/omega.jpg" alt="">	
+		        	</a>
+		        </div><!--END Onion Omega project-->
+		   		
+		        
+		         <!--Onion Omega project-->
+		        <div class='my-project'>
+		        	<div class="col-md-6"></div>
+		        	<a href="s" data-toggle="modal" data-target="#DriverDisplayModal">
+		        	<img class="img-fluid" src="images/display3.png" alt="">	
+		        	</a>
+		        </div><!--END Onion Omega project-->
+	        </div><!---END projects row2--->
+
+    	</div>
+
+
+<!--   <button class="collapsible">Open Collapsible</button>
+    <div class="cont"><a href="s" data-toggle="modal" data-target="#DriverDisplayModal">
+      <p>Lorem ipsum asdakdhaskjdhsakjhdjkashdkjahskdjhasjkdhkjashdjkhaskjdhaskjdhkajshdkjahjksdhsakjdhkjashdkjh kjhkj ...</p>
+    </div> -->
+
       </div>
+    </section><!---END PROJECTS---->
+
+    <!----MODAL POPUP--->
+
+    <section class="modal fade" id="DriverDisplayModal" aria-hidden="true">
+    	<div class="modal-dialog modal-lg modal-dialog-centered ">
+    		<div class="modal-content">
+    			<div class="modal-header">
+    				<button class="close" type="button" data-dismiss="modal">x</button>

```

Comment:
This button looks kinda ugly I don't think you really need it but maybe you can change it to an `up-arrow` or something